src/examples/EventGroupExample/GroupedEventProcessor.h

```diff
@@ -26,8 +26,9 @@ class GroupedEventProcessor : public JEventProcessor {
 
     void Process(const std::shared_ptr<const JEvent>& event) override {
 
+        std::unique_ptr<JBenchUtils> bench_utils = std::make_unique<JBenchUtils>(event->GetEventNumber(), NAME_OF_THIS);
```

Comment:
The constructor for JBenchUtils might be expensive, so I'd consider adding  a method to reset the seed rather than recreating the object on every event. If you make `bench_utils` be a member variable, you can construct it exactly once and then reset the seed on every event. Also, you don't need the unique_ptr here because ownership stays with the component and there's no inheritance hierarchy to watch out for.